### DISCUSSION TITLE
Is Hexa-SPI (HSPI) found in STM32U5 family supported?

### DISCUSSION THREAD

**Discussion by ivanwagner** [zephyrproject-rtos/zephyr#76347] [Q&A] [answered]
Dear all,

I looked within the dts structure of the STM32U5 family and cannot see any definition for the HSPI registers. Is someone working on it or will it be supported?

**Comment by FRASTM** [comment]
Hello @ivanwagner, 
Hexa-SPI is not supported by any stm32 device.
Moving to the zephyr MSPI is probably a interesting approach (instead of adding a new stm32 driver), but coming after single/quad/octo modes.

**Reply by ivanwagner** [reply]
Thank you for the answer. I took a look at MSPI. What about adding the missing registers for the HSPI peripheral on the STM32U5 device? That is something that needs to be done in any case, correct?

**Reply by FRASTM** [reply, marked as answer]
Sure, adding the hspi peripheral to the devices DTS and to the board, including the pin definition, is required in any case.
